src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -27,10 +31,23 @@ private Lift() {
         rightLiftMotor.setIdleMode(IdleMode.kCoast);
         rightLiftMotor.follow(leftLiftMotor);
 
+        leftServo.setBoundsMicroseconds(2000, 1800, 1500, 1200, 1000);
+        rightServo.setBoundsMicroseconds(2000, 1800, 1500, 1200, 1000);
+
         leftLiftMotor.burnFlash();
         rightLiftMotor.burnFlash();
     }
 
+    public void enableRatchets() {
+        leftServo.set(1);
+        rightServo.set(1);
+    }
+
+    public void disableRatchets() {
+        leftServo.set(0);
+        rightServo.set(0);
+    }
+
     public void runPos() {
         leftLiftMotor.set(LiftConstants.motorSpeed);
     }
```

Comment:
Set a constant for a negative speed instead of negating the value on line 56.

Reply:
2 different approaches, neither of them necessarily bad. If the pos and neg directions will always be the same power this is fine, but if theres even a change they could differ in the future I would split them into 2 constants